packages/manager/apps/container/src/container/nav-reshuffle/sidebar/tooltip/SidebarLinkTooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from "react";
+import { Tooltip } from "react-tooltip";
+
+type Props = { 
+    children: React.ReactNode, 
+    content: string 
+};
+
+export const ShortSidebarLinkTooltip: FunctionComponent<Props> = ({
+    content,
+    children
+}) => {
+
+    const id = 'SidebarLinkTooltip';
```

Comment:
Id should be unique

Reply:
There are two ways to use Tooltip with: anchor a unique id , and a version with a non-unique id see the documentation. i had used non unique id version. to avoid confusion, I switched to anchor. https://react-tooltip.com/docs/examples/basic-examples

Reply:
There are two ways to use Tooltip with: anchor a unique id , and a version with a non-unique id see the documentation https://react-tooltip.com/docs/examples/basic-examples. i had used non unique id version. to avoid confusion, I switched to anchor.